middleware.js

```diff
@@ -29,14 +64,18 @@ function clientErrorHandler (err, req, res, next) {
 
 
 function errorHandler (err, req, res, next) {
+  console.log("errorHandler, err.message: ", err.message);
```

Comment:
`console.log()` or should we use **mozlog** here?

Reply:
Good catch. We shouldn't actually need this `console.log` since `log.error` in `logErrors` above should take care of the logging.